gen3utils/etl/etl_validator.py

```diff
@@ -450,14 +450,12 @@ def validate_mapping(dictionary_url, mapping_file, manifest):
     tube_version = get_manifest_version(
         manifest["versions"], "tube", release_tag_are_branches=False, warn=False
     )
-    underscore = False
+    underscore = True
     if tube_version is not None and type(tube_version) != str:  # str if branch
-        if not is_release_tag(tube_version) and tube_version >= version.parse("0.4.0"):
-            underscore = True
-        elif tube_version >= version.parse("2020.10"):
-            underscore = True
-    if tube_version == "master":
-        underscore = True
+        if is_release_tag(tube_version) or tube_version < version.parse("0.4.0"):
```

Comment:
`if is_release_tag(tube_version): underscore = False` => why should we not use the underscore if it’s a release tag? the newer release tags need the underscore, right?

should it be this instead?
```
if tube_version < version.parse("0.4.0") or (is_release_tag(tube_version) and tube_version < version.parse("2020.10")):
    underscore = False
```